lisa/wlgen/rta.py

```diff
@@ -107,7 +109,7 @@ def run(self, cpus=None, cgroup=None, background=False, as_root=False):
         for task in self.tasks:
             # RT-app appends some number to the logs, so we can't predict the
             # exact filename
-            logfile = self.target.path.join(self.run_dir, '*{}*.log'.format(task))
```

Comment:
I'm not sure there's any value in splitting the `run_dir` and the `log_dir`. I can understand wanting to honor what's given in the rtapp conf, but even at the time of the above bug I'd say you were supposed to use https://github.com/ARM-software/lisa/blob/master/libs/wlgen/wlgen/rta.py#L237 rather than specify your own. 

My gripe with this mostly is about adding an extra parameter that will have abysmal uses (I know I'll never use it).

Reply:
If we expect to only do that with __LOGDIR__ placeholder, let's close that PR and #659 .